tensorflow_datasets/image/ham10000.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+import os
+import numpy as np
+import csv
+
+
```

Comment:
please run pylint (and cleanup unnecessary empty lines).

Reply:
I still see a lot of empty lines - did you run the pylint?

Reply:
I did run the pylint and removed empty lines,but after uploading ,the code seems to have empty lines